wiki/Submission/Beatmap_Ranking_Procedure/en.md

```diff
@@ -1,44 +1,23 @@
-Beatmap Ranking Procedure
-============================
+# Beatmap Ranking Procedure
 
-![Ranking flowchart.](Ranking_procedure.png "Ranking flowchart.")
+When a [beatmapset](/wiki/Glossary/#beatmapset) is first submitted, it is placed in the Pending or Work in Progress beatmap section. These maps are referred to as "unranked". Unranked maps do not have a scoreboard and move to the Graveyard section after a 28 day period of inactivity. The beatmapset can be moved to the Ranked section after going through the beatmap ranking process. This process involves receiving nominations and reaching the Qualified section.
 
-When a beatmap is first made, it is in the Pending or Work in Progress sections of the forum. These maps are referred to as "unranked". Unranked maps do not have a scoreboard and go to the Graveyard section of the forum after a period of inactivity. In order for a map to have a scoreboard and to be moved to the "Ranked" section of the forums, a map must go through the beatmap approval process. First it must get bubbled, and then ranked. Only BNs can bubble and subsequently rank a beatmap.
 
-Bubble ![Thinking icon](/wiki/shared/icon/bubble.gif "Thinking icon")
---------------------------------------------------------------------
+## Nominations 
 
-Before a beatmap can become ranked by a [BN](/wiki/People/Beatmap_Nomination_Group) (Beatmap Nominator), it must first get bubbled. In order for a [beatmap](/wiki/Beatmaps) to become bubbled, it must follow and be in accordance with the [ranking criteria](/wiki/Ranking_Criteria), and it must have at least 12 [Star Priority](/wiki/Glossary/#star-priority) (or SP). You can actively gain SP in two different ways. First, you can have others help mod your map. If you find their mod to be useful and give kudosu, you will gain 1 SP. You and/or others can also spend [kudosu](/wiki/Glossary/#kudosu) in order to "star" your map (A star costs 1 kudosu, and can be done through either the [mapper](/wiki/Glossary/#beatmapper)'s profile or through the [map](/wiki/Beatmaps)'s thread). Once a beatmap has at least 12 SP, a BN can bubble it if they feel it meets two conditions:
+In order for a beatmapset to receive a nomination, it must follow and be in accordance with the [ranking criteria](/wiki/Ranking_Criteria) and must have at least 5 hypes. Hypes can be obtained by having other users hype your beatmap in the Discussion page of the map, or under the map's description. Once a beatmap has at least 5 hypes, a BN can nominate it if they feel it meets two conditions:
```

Comment:
Ranking Criteria already will be listed as one of the BN's conditions in the bottom.
```suggestion
In order for a beatmapset to be eligible for a nomination, it must have at least 5 hypes. Hypes can be obtained by having other users hype your beatmap in the Discussion page of the map, or under the map's description. Once a beatmap has at least 5 hypes, a BN can nominate it if they feel it meets two conditions:
```